test/extensions/filters/network/common/fuzz/uber_filter.h

```diff
@@ -0,0 +1,98 @@
+#include "envoy/network/filter.h"
+
+#include "common/protobuf/protobuf.h"
+#include "common/singleton/manager_impl.h"
+
+#include "test/extensions/filters/common/ext_authz/mocks.h"
+#include "test/extensions/filters/network/common/fuzz/network_filter_fuzz.pb.validate.h"
+#include "test/fuzz/utility.h"
+#include "test/mocks/buffer/mocks.h"
+#include "test/mocks/network/mocks.h"
+#include "test/mocks/server/mocks.h"
+
+namespace Envoy {
+namespace Server {
+namespace Configuration {
+class FakeFactoryContext : public MockFactoryContext {
```

Comment:
Can this be separated out to a separate header in this directory? utils/fakes

Reply:
> Can this be separated out to a separate header in this directory? utils/fakes

Yes, I have separated them. Thanks for the suggestion!